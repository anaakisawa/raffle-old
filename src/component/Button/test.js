
import React from 'react';
import Button from './index';
import renderer from 'react-test-renderer';

describe('Component - Button', () => {
  test('renders a button element', () => {
    const component = renderer.create(
      <Button text="Go"/>,
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
});


/* */
