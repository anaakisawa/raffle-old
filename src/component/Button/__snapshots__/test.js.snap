// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - Button renders a button element 1`] = `
<button
  className="Button"
>
  Go
</button>
`;
